impeller/display_list/display_list_dispatcher.cc

```diff
@@ -838,8 +874,10 @@ void DisplayListDispatcher::drawAtlas(const sk_sp<flutter::DlImage> atlas,
                                       flutter::DlImageSampling sampling,
                                       const SkRect* cull_rect,
```

Comment:
I think we need to incorporate this cull rect at some point, but we can do this as follow-up work this can be done by either surrounding DrawAtlas with DrawClip/Restore calls or doing clipping tricks in the draw call + coverage itself.

Reply:
I added a todo is aiks canvas.